src/sentry/api/bases/organization_events.py

```diff
@@ -260,6 +261,25 @@ def save_split_decision(self, widget, has_errors, has_transactions_data):
 
         return new_discover_widget_split
 
+    def save_discover_saved_query_split_decision(self, query, has_errors, has_transactions_data):
+        """This can be removed once the discover dataset has been fully split"""
+        if has_errors and not has_transactions_data:
+            decision = DiscoverSavedQueryTypes.ERROR_EVENTS
+        elif not has_errors and has_transactions_data:
+            decision = DiscoverSavedQueryTypes.TRANSACTION_LIKE
+        else:
+            # In the case that neither or both datasets return data,
+            # we don't split this yet and can make multiple queries to check each time.
+            # This will help newly created widgets or infrequent count
+            # widgets that shouldn't be prematurely assigned a side.
+            decision = DiscoverSavedQueryTypes.DISCOVER
+        sentry_sdk.set_tag("discover_split_decision", decision)
```

Comment:
nit, can we call this
```suggestion
        sentry_sdk.set_tag("discover.split_decision", decision)
```
so the ui does that thing where we organize this tag with the other discover tags